easybuild/easyconfigs/m/Metal/Metal-2020-05-05-foss-2020b.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'CMakeMake'
+
+name = 'Metal'
+version = '2020-05-05'
+
+homepage = 'http://csg.sph.umich.edu/abecasis/Metal/'
+description = """Metal - Meta Analysis Helper. The METAL software is designed to facilitate meta-analysis of large
+ datasets (such as several whole genome scans) in a convenient, rapid and memory efficient manner."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
```

Comment:
@fizwit any thoughts on whether this could be moved down to `GCC`?

Reply:
compiles and runs with GCC 10.2.0, updating PR

Reply:
You missed removing the foss variant, so now there are 2 easyconfigs in this PR. `eb --update-pr 13485 :Metal-2020-05-05-foss-2020b.eb` (b.t.w I recommend that you don't specify `--pr-title`  on new PR's  unless you are only changing something